filters/auth/forwardtokenpart.go

```diff
@@ -0,0 +1,74 @@
+package auth
+
+import (
+	"fmt"
+
+	"github.com/zalando/skipper/filters"
+	"golang.org/x/net/http/httpguts"
+)
+
+const (
+	ForwardTokenPartName = "forwardTokenPart"
+)
+
+type (
+	forwardTokenPartSpec   struct{}
+	forwardTokenPartFilter struct {
+		HeaderName string
+		Key        string
+	}
+)
+
+// NewForwardTokenPart creates a filter to forward part of the result of token info or
+// token introspection or to the backend server.
```

Comment:
>...token introspection or <?> to the backend server.

Something seems to be missing at `<?>`. Also we would need to update filter docs similar to https://github.com/zalando/skipper/blob/master/docs/reference/filters.md#forwardtoken